core/common/hexutil.cpp

```diff
@@ -3,27 +3,31 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <algorithm>
-#include <boost/algorithm/hex.hpp>
+#include "common/hexutil.hpp"
 
```

Comment:
The same about empty line

Reply:
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes